Makefile

```diff
@@ -5,6 +5,7 @@ ANSIBLE_VERBOSE?=
 ANSIBLE_LIMIT?=all:!localhost
 # Extra flags to pass to Ansible (e.g. --skip-tags filebeat).
 ANSIBLE_EXTRA_FLAGS?=
+GROUPS?=centos debian sles windows
```

Comment:
Can you add a description like these are the Ansible host groups declared in the `hosts` file.